user_manual/groupware/mail.rst

```diff
@@ -107,6 +107,12 @@ Mail can move a message to a different folder when it is marked as junk.
 
 .. figure:: images/mail_move-message-to-junk-folder.png
 
+Refresh mailbox
```

Comment:
I guess the page needs a general overhaul because here we say *mailbox* (IMAP terminology) and below it says *folder* (user interface terminology). This creates a bit of ambiguity.